libs/components/molecules/src/lib/footer/footer.component.html

```diff
@@ -0,0 +1,72 @@
+<!-- need to chnage the links -->
+<div class="footer">
+  <div class="container">
+    <form action="http://localhost:4400/?path=/story/footercomponent--primary" method="get" target="_blank">
+      <button class="button" mat-stroked-button>
+        <mat-icon>info</mat-icon>
+        Ilustration
+      </button>
+    </form>
+
+    <div class="dropup">
+      <button class="button" mat-stroked-button><mat-icon>file_download</mat-icon>Download</button>
+      <div class="dropup-content">
+        <a>Download XLXS</a>
+        <a>Download JSON-LD</a>
+        <a>Download AI</a>
+        <a>Download PDF</a>
+        <a>Download PNG</a>
+
+        <!-- <mat-select>
+  <mat-option *ngFor="let item of items$ | async" class="option" [value]="item">
+    {{ item.label }}
+  </mat-option>
+</mat-select> -->
+
+        <!-- <mat-select>
+    <mat-option value="xlxs">Download XLXS</mat-option>
+    <mat-option value="joson">Download JSON-LD</mat-option>
+    <mat-option value="pdf">Download PDF</mat-option>
+    <mat-option value="ai">Download AI</mat-option>
+    <mat-option value="png">Download PNG</mat-option>
+  </mat-select> -->
+      </div>
+    </div>
+    <form action="http://localhost:4400/?path=/story/footercomponent--primary" method="get" target="_blank">
+      <button class="button" mat-stroked-button><mat-icon>code</mat-icon> Embed</button>
+    </form>
+
+    <div class="end">
```

Comment:
Use a filler element to move things to the end https://github.com/hubmapconsortium/hra-ui/wiki/Responsive-design